openfga/v1/openfga_service.proto

```diff
@@ -825,9 +915,9 @@ message WriteRequest {
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"01YCP46JKYM8FJCQ37NMBYHE5X\""}
   ];
 
-  openfga.v1.TupleKeys writes = 2;
+  WriteRequestTupleKeys writes = 2;
 
-  openfga.v1.TupleKeys deletes = 3;
+  WriteRequestTupleKeys deletes = 3;
```

Comment:
The structure of tuples to be deleted differs from the structure of those to be written. When deleting a tuple you simply provide `(object, relation, user)` (e.g. the PK). When writing a tuple though, then you can provide up to `(object, relation, user, condition_name, condition_context)`. So this API change is a bit clunky.

Reply:
Why do we want it to be different?  Let's say that there are two tuples, one with condition and one without.  How can we delete only the no codition?

Reply:
@adriantam `(object, relation, user)` is the primary key. We don't allow the same tuple to be written more than once. A tuple is either conditioned or not, but not both at the same time. So from a delete perspective it only makes sense to accept a `(object, relation, user)` input for deletion. For writes you can provide an optional condition and condition context, but for deletes it is irrelevant.